src/EditorFeatures/Core/FindUsages/IDefinitionsAndReferencesFactory.cs

```diff
@@ -193,7 +193,7 @@ internal static class DefinitionItemExtensions
             var documentSpan = await ClassifiedSpansAndHighlightSpanFactory.GetClassifiedDocumentSpanAsync(
                 document, sourceSpan, cancellationToken).ConfigureAwait(false);
 
-            return new SourceReferenceItem(definitionItem, documentSpan, referenceLocation.SymbolUsageInfo);
+            return new SourceReferenceItem(definitionItem, documentSpan, referenceLocation.SymbolUsageInfo, referenceLocation.CustomColumns);
```

Comment:
>CustomColumns [](start = 126, length = 13)

Probably better to rename this as `CustomProperties` or `CustomAttributes`. Columns is not something intuitive for a `SourceReferenceItem`

Reply:
Maybe even `AdditionalProperties`

---
In reply to: [312660643](https://github.com/dotnet/roslyn/pull/37802#discussion_r312660643) [](ancestors = 312660643)